src/main/java/gov/cms/madie/services/TranslationServiceClient.java

```diff
@@ -68,4 +70,25 @@ public CqlLookups getCqlLookups(QdmMeasure measure, String accessToken) {
       throw new TranslationServiceException(msg, ex);
     }
   }
+
+  public List<SourceDataCriteria> getRelevantDataElements(Measure measure, String accessToken) {
+    URI uri =
+        URI.create(
+            translatorClientConfig.getBaseUrl() + translatorClientConfig.getRelevantDataElements());
+    HttpHeaders headers = new HttpHeaders();
+    headers.set(HttpHeaders.AUTHORIZATION, accessToken);
+    HttpEntity<Measure> entity = new HttpEntity<>(measure, headers);
+    ParameterizedTypeReference<List<SourceDataCriteria>> responseType =
+        new ParameterizedTypeReference<>() {};
+    try {
+      log.info("fetching the relevant data elements for measure");
```

Comment:
can we add the measure ID to this log statement so we have a little more context if troubleshooting

Reply:
fixed